include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -375,6 +375,8 @@ struct __brick_copy_n<__hetero_tag<_BackendTag>, _ExecutionPolicy>
     {
         __target = ::std::forward<_SourceT>(__source);
     }
+
+    _ONEDPL_IS_DEVICE_COPYABLE()
```

Comment:
Unnecessary, as mentioned above.  I think `__brick_fill_n` may be the one you intended to add to.